core/arch/arm/kernel/generic_boot.c

```diff
@@ -73,6 +73,9 @@
  */
 #define PADDR_INVALID		ULONG_MAX
 
+static uint8_t pseudo_device_id[160] __early_bss __attribute__ ((aligned (8)));
```

Comment:
At one point it was cast to uint32_t.... that went away before the last push.  I removed it.